shell/assets/translations/en-us.yaml

```
@@ -5619,6 +5625,10 @@ workload:
     noPorts: There are no ports configured.
     noServiceAccess: You do not have permission to create or manage services
     ports:
+      expose: Networking
+      description: 'Define a Service to expose the container, or define a non-functional, named port so that humans will know where the app within the container is expected to run.'
+      detailedDescription: If ClusterIP, LoadBalancer or NodePort is selected, a Service is automatically created that will select the Pods in this workload using labels.
```

```suggestion
      detailedDescription: If ClusterIP, LoadBalancer, or NodePort is selected, a Service is automatically created that will select the Pods in this workload using labels.
```

I prefer the Oxford comma